src/setupProxy.js

```diff
@@ -0,0 +1,7 @@
+module.exports = (app) => {
```

Comment:
Is this required?

Reply:
While using web assembly code we required this one to add for security purpose